Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Use this with
+#
+# docker build -t canisters .
+# docker run --rm --entrypoint cat canisters /ic-btc-canister.wasm > ic-btc-canister.wasm
+# docker run --rm --entrypoint cat canisters /uploader-canister.wasm > uploader-canister.wasm
+
+# The docker image. To update, run `docker pull ubuntu` locally, and update the
+# sha256:... accordingly.
+FROM ubuntu@sha256:626ffe58f6e7566e00254b638eb7e0f3b11d4da9675088f4781a50ae288f3322
+
+ARG rust_version=1.62.1
```

Comment:
nit: can we add a note here that, if we update this version, we should also update the version in `rust-toolchain.toml`? And vice versa in the `rust-toolchain.toml` file as well.

Reply:
Done.